reflex/utils/exec.py

```diff
@@ -489,35 +491,5 @@ def is_prod_mode() -> bool:
     Returns:
         True if the app is running in production mode or False if running in dev mode.
     """
-    current_mode = os.environ.get(
-        constants.ENV_MODE_ENV_VAR,
-        constants.Env.DEV.value,
-    )
-    return current_mode == constants.Env.PROD.value
-
-
-def is_frontend_only() -> bool:
```

Comment:
are these functions moved somewhere else? or just removed?

if removed, I think it would be prudent to deprecate them and remove them completely in 0.7.0 instead of in a 0.6.x dot release

Reply:
I just removed them because they just added another indirection which did nothing. If you prefer i can just readd them or deprecate them.